File: backend/components/metiers/administration_publique/guides/core/fixtures/samples/sample_fixture.test.py
"""
Test unitaire pour sample_fixture.py (guides/core/fixtures)
"""
from .sample_fixture import sample_guide_fixture

def test_sample_guide_fixture():
    fixture = sample_guide_fixture()
    assert isinstance(fixture, dict)
    assert fixture["name"] == "Sample Guide Fixture"
    assert fixture["type"] == "guide"
    assert fixture["status"] == "active"

File: backend/components/metiers/administration_publique/guides/core/fixtures/samples/sample_fixture.test.js
// sample_fixture.test.js - Test unitaire pour sample_fixture.js (guides/core/fixtures)
const sampleFixture = require('./sample_fixture');
describe('guides/core/fixtures/sample_fixture.js', () => {
  it('doit exposer un objet sample correct', () => {
    expect(sampleFixture.sample).toBeDefined();
    expect(sampleFixture.sample.name).toBe('Sample Guide Fixture');
    expect(sampleFixture.sample.type).toBe('guide');
    expect(sampleFixture.sample.status).toBe('active');
  });
});
